charts/k8ssandra/values.yaml

```diff
@@ -48,6 +66,9 @@ cassandra:
       pullPolicy: IfNotPresent
   # -- The jmx-credentials init container that configures JMX credentials.
   jmxCredentialsConfig:
+    # -- Security context override for jmx init container
+    # K8ssandra defaults readOnlyRootFilesystem: true
+    securityContext: {}
```

Comment:
Are there security settings that won't work with this container? If so we should either document it or not allow it. I'm fine with just documenting it and doing that in a follow PR if that works for you.